<ui version="4.0" >
 <class>BIGHOME_RadioClass</class>
 <widget class="QWidget" name="BIGHOME_RadioClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>410</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize" >
   <size>
    <width>432</width>
    <height>510</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>BIGHOME_Radio</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2" >
   <property name="leftMargin" >
    <number>0</number>
   </property>
   <property name="topMargin" >
    <number>0</number>
   </property>
   <property name="rightMargin" >
    <number>0</number>
   </property>
   <property name="bottomMargin" >
    <number>20</number>
   </property>
   <property name="horizontalSpacing" >
    <number>0</number>
   </property>
   <property name="verticalSpacing" >
    <number>10</number>
   </property>
   <item row="0" column="0" >
    <spacer name="horizontalSpacer_11" >
     <property name="orientation" >
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0" >
      <size>
       <width>325</width>
       <height>24</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" >
    <layout class="QHBoxLayout" name="horizontalLayout_5" >
     <property name="spacing" >
      <number>10</number>
     </property>
     <item>
      <widget class="QPushButton" name="s" >
       <property name="minimumSize" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text" >
        <string>S</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="c" >
       <property name="minimumSize" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text" >
        <string>C</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="x" >
       <property name="minimumSize" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize" >
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="cursor" >
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="text" >
        <string>X</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2" >
    <layout class="QGridLayout" name="gridLayout" >
     <item row="0" column="0" >
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <property name="spacing" >
        <number>20</number>
       </property>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4" >
         <item>
          <spacer name="horizontalSpacer_4" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>17</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="photoLb" >
           <property name="minimumSize" >
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>200</width>
             <height>200</height>
            </size>
           </property>
           <property name="text" >
            <string>photo</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3" >
           <property name="orientation" >
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0" >
            <size>
             <width>18</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" >
         <property name="spacing" >
          <number>10</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" >
           <item>
            <spacer name="horizontalSpacer" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>28</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="name" >
             <property name="minimumSize" >
              <size>
               <width>170</width>
               <height>30</height>
              </size>
             </property>
             <property name="maximumSize" >
              <size>
               <width>170</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet" >
              <string notr="true" >color: rgb(255, 255, 255);
font: 75 15pt "微软雅黑";</string>
             </property>
             <property name="text" >
              <string>光辉岁月-Beyond</string>
             </property>
             <property name="alignment" >
              <set>Qt::AlignCenter|Qt::AlignHCenter|Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2" >
             <property name="orientation" >
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0" >
              <size>
               <width>28</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWidget" native="1" name="widget" >
           <property name="minimumSize" >
            <size>
             <width>256</width>
             <height>80</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>256</width>
             <height>80</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="1" column="0" >
      <layout class="QHBoxLayout" name="horizontalLayout_2" >
       <property name="spacing" >
        <number>0</number>
       </property>
       <property name="leftMargin" >
        <number>50</number>
       </property>
       <property name="topMargin" >
        <number>0</number>
       </property>
       <property name="rightMargin" >
        <number>50</number>
       </property>
       <property name="bottomMargin" >
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="time1" >
         <property name="minimumSize" >
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>30</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet" >
          <string notr="true" >color: rgb(16, 61, 99);
font: 8pt "微软雅黑";</string>
         </property>
         <property name="text" >
          <string>3：28</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="timerSlider" >
         <property name="minimumSize" >
          <size>
           <width>240</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>240</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="time2" >
         <property name="minimumSize" >
          <size>
           <width>35</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>35</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet" >
          <string notr="true" >color: rgb(16, 61, 99);
font: 8pt "微软雅黑";</string>
         </property>
         <property name="text" >
          <string>-1：10</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" >
      <layout class="QHBoxLayout" name="horizontalLayout" >
       <property name="spacing" >
        <number>10</number>
       </property>
       <property name="leftMargin" >
        <number>110</number>
       </property>
       <property name="rightMargin" >
        <number>110</number>
       </property>
       <item>
        <widget class="QPushButton" name="play" >
         <property name="minimumSize" >
          <size>
           <width>45</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>45</width>
           <height>45</height>
          </size>
         </property>
         <property name="cursor" >
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text" >
          <string>play</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="next" >
         <property name="minimumSize" >
          <size>
           <width>45</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>45</width>
           <height>45</height>
          </size>
         </property>
         <property name="cursor" >
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text" >
          <string>next</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="like" >
         <property name="minimumSize" >
          <size>
           <width>45</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>45</width>
           <height>45</height>
          </size>
         </property>
         <property name="text" >
          <string>like</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deletes" >
         <property name="minimumSize" >
          <size>
           <width>45</width>
           <height>45</height>
          </size>
         </property>
         <property name="maximumSize" >
          <size>
           <width>45</width>
           <height>45</height>
          </size>
         </property>
         <property name="text" >
          <string>delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>horizontalSpacer_11</zorder>
  <zorder></zorder>
  <zorder></zorder>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
